values.yaml

```diff
@@ -211,6 +217,16 @@ mastodon:
     workers: "2"
     persistentTimeout: "20"
 
+  # Additional settings for the cache buster.
+  cacheBuster:
```

Comment:
We may want a link to the documentation for this feature, as the format is specific

Reply:
Where is the documentation for this stuff specifically? I looked myself and couldn't find more details about mastodon's cache buster specifically.

Reply:
We do not seem to have a documentation on this…

Reply:
I added documentation in this PR: https://github.com/mastodon/documentation/pull/1383

Once it is merge, can you update this to include a link to the doc?

Reply:
Awesome! Added links in the comments.